izpack-maven-plugin/src/main/java/org/izpack/mojo/IzPackNewMojo.java

```diff
@@ -224,13 +261,22 @@ private void initMavenProperties(PropertyManager propertyManager)
                 }
                 if (!containsExcludedProperty(propertyName, exclusionList))
                 {
-                    if (propertyManager.addProperty(propertyName, value))
+                    String existingValue = propertyManager.getProperty(propertyName);
+                    if (existingValue != null && existingValue.equals(value))
                     {
-                        getLog().debug("Maven property added: " + propertyName + "=" + value);
+                        getLog().debug("Maven property exists: " + propertyName + "=" + value);
                     }
                     else
                     {
-                        getLog().warn("Maven property " + propertyName + " could not be overridden");
+                        if (propertyManager.addProperty(propertyName, value))
```

Comment:
Here I think using an other `else if` block would be easier to read

Reply:
Note that I tested the plugin by building the demo app.
The unit tests for the plugin currently fail as default behaviour has changed when a finalName and classifier are provided.
I'm not sure how to correct the unit tests (or whether the default behaviour should be corrected to meet the current unit tests).

Reply:
Done